membership/src/main/scala/zio/membership/Membership.scala

```diff
@@ -7,34 +7,41 @@ import zio.stream._
  * The main entrypoint to the membership library. Allows querying the cluster state
  * and sending messages to members.
  */
-trait Membership {
-  val membership: Membership.Service[Any]
+trait Membership[T] {
```

Comment:
Nitpick: A instead of T :).

Reply:
probably makes sense. I've tried to be consistent with T representing the node identifier, but probably it does not buy much